SPECS/golang/golang.spec

```diff
@@ -144,6 +144,9 @@ fi
 %{_bindir}/*
 
 %changelog
+* Wed Apr 03 2024 Davis Goodin <dagood@microsoft.com> - 1.21.9-1
```

Comment:
With a `go` update, we also bump the release tag of packages that depend on `go`

Reply:
Yeah, we're still waiting on a script to be able to reasonably do that. https://github.com/microsoft/azurelinux/pull/8317#issuecomment-1986215104